modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogServiceImpl.java

```diff
@@ -137,66 +137,61 @@ public CatalogServiceImpl(UpdateLog updateLog, HybridClock clock, long delayDura
         this.delayDurationMs = delayDurationMs;
     }
 
-    /** {@inheritDoc} */
```

Comment:
what is wrong with this javadoc (and rest of the similar)?

Reply:
They are useless, if you open the documentation, then the parent is automatically loaded when reading from the development environment.